renovate.json

```diff
@@ -3,15 +3,15 @@
     "extends": [
       "config:base"
     ],
-    "baseBranches": ["main", "release-2.12", "release-2.11", "release-2.10"],
```

Comment:
Did we changed recently? I think we agreed to support last 2 minor version, so here should be ["main", "release-2.12","release-2.11"],

Reply:
the release process says to do this - 3 versions + main until 2.13 is released. After that delete the latest release. https://github.com/grafana/mimir/issues/8390. I will clean this up once the release finishes